### DISCUSSION TITLE
font-weight for h1 and h2

### DISCUSSION THREAD

**Discussion by KaiWaldrant** [quarto-dev/quarto-cli#5425] [Q&A] [answered]
I noticed that since the release of quarto v1.3 the `font-weight` of the h1 and h2 went from `500` to `600`. It is unclear where and how this is done because we want to revert this. Can someone shed some light on this for me ? atm I am just changing this in a custom css file.

**Comment by mcanouil** [comment, marked as answer]
Could you give more details? In which case exactly? I figure it's HTML, but book? website? epub? html single document?

Note that knowing the where/when (I figure it should be this cbda21d70494be3ffca14fce046ade4403bed9c6 but without details I might be wrong) does not really matter as the only way to "revert" is to overwrite with css/scss.

**Reply by KaiWaldrant** [reply]
Its for html, I always forget to add that part.
But you have given me the correct commit.

**Reply by cderv** [reply]
@dragonstyle FYI an other change in cbda21d70494be3ffca14fce046ade4403bed9c6 that impacted user's website. 

Maybe we should / could document style change better, or use sass variables with new default to simplifies revert by only setting a variable. 
Style change are always opinionated and we'll be also impacted by Bootstrap update probably. I just noticed it is the second time, so just want to share the thoughts if we can consider improving our communication or choices in that matter.

**Reply by KaiWaldrant** [reply]
@cderv I agree with you. We actually want to revert some style changes made in quarto v1.3 but it was hard to find what changed exactly.

I/we personally found the change of the font-weight a huge impact and would rather see this as an option then as default.

**Reply by dragonstyle** [reply]
@cderv Agree that we should've documented this better for sure! 

In this case, a font change like this is something I think we should be free to change between versions, though we should document it. 

It will be hard to create a style that doesn't have any opinion, so we'll always choose a presentation that we think works the best and do our best to allow it be customized and changed. This should be straightforward using CSS (as it was before if you thought our heading fonts needed to be heavier!). I think there is a tension between making the number of options grow to excess and simply saying use CSS - in this case I think CSS seems like a reasonable solution.

**Reply by mcanouil** [reply]
For what's worth, if the CSS changes are documented, it is pretty straightforward to update your own website/document css/scss file to include a "fix".
I also don't believe all style changes should be optional, the main reason is that Quarto relies on Bootswatch/Bootstrap themes which do not provide option to go back to an old version, thus for me it does not make any sense to even try it (it's kind of doomed to fail already).
This said, for things that can't be reverted simply with partials and/or (s)css, an option seems needed.

**Reply by cderv** [reply]
> I think there is a tension between making the number of options grow to excess and simply saying use CSS - in this case I think CSS seems like a reasonable solution.

Indeed - growing list of options and variables could make it hard to follow and know also what to tweak. Probably for next release we should have a dedicated news docs about our change in default style - that is what I take from this.
